muselog/flask.py

```diff
@@ -0,0 +1,46 @@
+"""
+Custom logger for user auth service in python flask 
+"""
+
+import sys
+import logging
+
+from time import strftime
+from muselog import add_gelf_handler
+
+logger = logging.getLogger(__name__)
+
+# http://flask.pocoo.org/docs/0.12/api/#flask.Flask.after_request
+def log_request(app, request, response):
+    """Logging after every flask request with extra context for use w/ Graylog-enabled apps.
+    
+    Usage:
+    @app.after_request
+    def after_request(response):
+        return muselog.flask.log_request(app, request, response)
+
+    """
+
+    # http://flask.pocoo.org/docs/0.12/quickstart/#logging
+    if response.status_code < 400:
+        log_method = app.logger.info
```

Comment:
In my local changes I removed the use of `app.logger...` and changed it to match the format in `tornado.py`. So it should just be `logger.info`

Reply:
This is because we setup the default python logger to have a GELF TLS handler in `setup.py` and in order for that log to be processed by the TLS handler we need to use the python `logger` not the flask app logger